biome.json

```diff
@@ -1,41 +1,45 @@
 {
-  "formatter": {
-    "enabled": true,
-    "formatWithErrors": false,
-    "ignore": [],
-    "attributePosition": "auto",
-    "indentStyle": "space",
-    "indentWidth": 4,
```

Comment:
Isn't javascript "usually" indented with 2 spaces?

Reply:
No strong opinion on my side, let's switch to 2 👍